src/styles/colors.modules.scss

```diff
@@ -0,0 +1,32 @@
+// background-and-text-colors
+$main1: #FEF9EC;
```

Comment:
HI!
We use native css variables.
Instead of `$main1: #FEF9EC;` =>  `--main1: #FEF9EC;`